common/src/main/java/tachyon/util/io/FileUtils.java

```diff
@@ -181,4 +181,24 @@ public static void createFile(File file) throws IOException {
       throw new IOException("File already exists " + file.getPath());
     }
   }
+
+  /**
+   * Creates an empty directory and its intermediate directories if necessary.
+   *
+   * @param file the file to create
+   * @throws IOException if an I/O error occurred or file already exists
+   */
+  public static void createDir(File file) throws IOException {
```

Comment:
I think this should be a string parameter, to stay with convention (in a different PR)